pkgs/development/python-modules/wsgi-intercept/default.nix

```
@@ -36,6 +41,8 @@ buildPythonPackage rec {
   ];
 
   disabledTests = [
+    "test_urllib3"
```

A comment would be helpful for the future. Some like:

```suggestion
    # Tests require network access
    "test_urllib3"
```

Or if possible the actual issue.

---

ok, will do.